schemas/v3.1/meta-openapi.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft/2019-WIP/schema#",
+  "$id": "https://spec.openapis.org/oas/3.1/meta",
+  "$vocabulary": {
+    "https://specs.openapis.org/oas/3.1/vocab/openapi": true
+  },
+  "$recursiveAnchor": true,
+  "title": "OpenAPI Vocabulary Schema",
+  "type": ["object", "boolean"],
+  "allOf": [
+    {
+      "$ref": "http://json-schema.org/draft/2019-WIP/schema"
```

Comment:
You don't actually want this here- a vocabulary meta-schema should only describe itself.  You still want `recursiveAnchor` the way you have it a few lines up because that ensures that this vocabulary meta-schema can be combined and extended properly, but you don't want the `allOf`.  Note that there is no `allOf` in http://json-schema.org/work-in-progress/meta/WIP-validation.json

You will need a separate meta-schema that combines this vocabulary with the standard ones and serves as the meta-schema for OAS Schema Objects.  Then you will reference that meta-schema wherever you want to allow schemas that use all of the combined vocabularies.  That separate meta-schema will look almost identical to http://json-schema.org/work-in-progress/WIP-schema.json except with the OAS vocabulary added to `$vocabularies`.  So you'll have seven entries in `$vocabularies` instead of six.